python/dgl/nn/pytorch/conv/gcn2conv.py

```diff
@@ -256,29 +256,12 @@ def forward(self, graph, feat, feat_0, edge_weight=None):
             feat_0 = feat_0[: feat.size(0)] * self.alpha
 
             if self._project_initial_features:
-                rst = feat.add_(feat_0)
-                rst = th.addmm(
-                    feat,
-                    feat,
-                    self.weight1,
-                    beta=(1 - self.beta),
-                    alpha=self.beta,
-                )
+                rst = feat + feat_0
```

Comment:
To keep consistency?
```suggestion
                feat_sum = feat + feat_0
```